x-pack/platform/plugins/shared/onechat/public/application/components/tools/form/validation/index_search_tool_form_validation.ts

```diff
@@ -85,13 +34,11 @@ const indexSearchFormValidationSchema = (toolsService: ToolsService) =>
     pattern: z
       .string()
       .min(1, { message: indexSearchI18nMessages.pattern.requiredError })
-      .regex(/^.+[^,]$/, { message: indexSearchI18nMessages.pattern.trailingCommaError })
```

Comment:
this was mishandling `*`, and the trailing comma is handled by next check anyway